horovod/common/operations.cc

```diff
@@ -1587,6 +1609,13 @@ void InitializeHorovodOnce(const int* ranks, int nranks) {
   while (!horovod_global.initialization_done) {
     std::this_thread::sleep_for(std::chrono::milliseconds(1));
   }
+
+  LOG(DEBUG) << "Background thread init done";
+#if HAVE_MLSL
```

Comment:
Can you leave a comment explaining what this signaling in the context of MLSL, and why we only log it when using MLSL?

Reply:
I guess, it's better to be removed.